### DISCUSSION TITLE
v3.1.0 - 插件端最终版发布

### DISCUSSION THREAD

**Discussion by Steve-xmh** [amll-dev/applemusic-like-lyrics#235] [Announcements]
历经 140+ 个提交后，我们终于推出了插件端的最终版本！

#### 完全重构的流体背景！
在和 Apple Music 几乎别无二致的情况下大幅度降低了性能开销！
在 BNCM 3.0.0+ 环境中，还能根据音频的鼓点动态变化哦！
在此非常感谢 @Eplorr 在背景着色器方面的贡献！

#### 新的渐变过渡算法！
使用 Web Animation API 重构了渐变过渡算法，现在单词的渐变过渡更加平滑且流畅了！

#### 新的辉光强调效果！
在效果和性能上更优于之前的版本了！

#### 新的可调节选项！
现在你可以设置更多的选项了：

- 单独隐藏音量控制条
- 悬浮到专辑图隐藏光标
- 禁用混色效果
- 加载音乐后自动暂停播放（便于录制视频）
- ...还有更多！

## 今后的计划

出于各方考虑，我们将终止 BNCM 插件版本的开发，故将这个版本定义为最终版本。
也因此，今后不再接受任何有关 BNCM 插件版本的反馈。

今后原 BNCM 插件中的代码将会被拆解并重新组合成一个便于二次开发的 React 组件库包，方便更多前端开发者使用，尽请期待！

同时，我们也将会完善外置播放器的功能，有望将其转型为本地播放器并继续提供协议连接播放功能，尽请期待！

<hr /><em>This discussion was created from the release <a href='https://github.com/Steve-xmh/applemusic-like-lyrics/releases/tag/v3.1.0'>v3.1.0 - 插件端最终版发布</a>.</em>

**Comment by oldeightLB** [comment]
完结撒花（）

**Comment by FriskTu** [comment]
感谢大佬
